grafana/knative-scaling.json

```diff
@@ -1,10 +1,8 @@
 {
-    "title": "Knative Serving - Scaling Debugging",
-    "uid": "Bn1mdAR7z",
     "__inputs": [
       {
-        "name": "DS_PROMETHEUS",
-        "label": "Prometheus",
+        "name": "DS_THANOS",
```

Comment:
Why the change to DS_THANOS is required? This chbage will make this dashboard inconsistent with the other ones in the repo that use DS_PROMETHEUS

Reply:
It is not intentional. There seems to be bug in grafana when it changes that for some reason.

Reply:
This is fixed now.